src/main/java/org/team1540/robot2018/commands/groups/IntakeSequence.java

```diff
@@ -3,13 +3,10 @@
 import edu.wpi.first.wpilibj.command.CommandGroup;
 import org.team1540.robot2018.commands.intake.AutoIntake;
 
-/**
- * Lower wrist > lower intake > intake a cube > move wrist up
- */
 public class IntakeSequence extends CommandGroup {
```

Comment:
This can actually just be removed and replaced with AutoIntake calls

Reply:
True, but we might want a sequence in the future so I think it's fine to leave it as a command group?